lib/client/identityfile/identity.go

```diff
@@ -614,10 +614,14 @@ func KeyFromIdentityFile(identityPath, proxyHost, clusterName string) (*client.K
 // is also added to its profile store with the limited profile data available
 // in the identity file.
 //
-// Since identity files do not save a proxy address, proxyAddr must be provided
-// to fill in this data gap. clusterName can also be provided to aim the key at
-// a leaf cluster rather than the default root cluster.
+// Use [proxyAddr] to specify the host:port-like address of the proxy.
+// This is necessary because identity files do not store the proxy address.
+// Additionally, the [clusterName] argument can ve used to target a leaf cluster
+// rather than the default root cluster.
 func NewClientStoreFromIdentityFile(identityFile, proxyAddr, clusterName string) (*client.Store, error) {
+	if proxyAddr == "" {
+		return nil, trace.BadParameter("missing proxyAddr loading identityFile. Login, configure a proxy address, or pass --proxy on the command line.")
```

Comment:
```suggestion
		return nil, trace.BadParameter("missing proxy address loading identityFile. Log in, configure a proxy address, or pass --proxy on the command line.")
```

- Since the user will see this message, use proper words and not variable names
- nit: login is a noun, I think you mean to say "log in"

Reply:
Excellent feedback. Integrating...